src/components/pages/Home/RenderHomePage.js

```diff
@@ -14,14 +14,9 @@ function RenderHomePage() {
   // useState for axios errors
   const [error, setError] = useState('');
   const [cityData, setCityData] = useState([]);
-  const [noSearch, setSearched] = useState([]);
   const [compareList, setCompareList] = useState({
-    city1: '',
-    state1: '',
-    city2: '',
-    state2: '',
-    city3: '',
-    state3: '',
+    cities: [],
```

Comment:
Great refactor. Converted 6 lines into 1 line by storing cities to compare in an array with a cities key